docs/components/connectors/protocol/http-webhook.md

```diff
@@ -27,14 +27,32 @@ Please refer to the [update guide](/guides/update-guide/connectors/060-to-070.md
 ![HTTP Webhook prefilled](../img/use-inbound-connector-template-filled.png)
 
 1. In the **Webhook Configuration** section, configure the **Webhook ID**. By default, **Webhook ID** is pre-filled with a random value. This value will be part of the Webhook URL. You will find more details about HTTP Webhook URLs [below](#activate-the-http-webhook-connector-by-deploying-your-diagram).
-2. Configure [HMAC authentication](https://en.wikipedia.org/wiki/HMAC) if required.
+2. Configure [HMAC authentication](https://en.wikipedia.org/wiki/HMAC) if required in the **Authentication** section.
 
 - Set the HMAC shared secret key which is used to calculate the message hash. The value is defined by a webhook administrator.
 - Set the HMAC header whose value contains an encrypted hash message. The exact value is provided by the external caller.
 - Select HMAC hash algorithm. The exact value is provided by the external caller.
 
-3. Configure **Activation Condition**. For example, given external caller triggers a webhook endpoint with the body `{"id": 1, "status": "OK"}`, the **Activation Condition** value might look like `=(request.body.status = "OK")`. Leave this field empty to trigger your webhook every time.
-4. Use **Variable Mapping** to map specific fields from the request into process variables using [FEEL](/components/modeler/feel/what-is-feel.md).
+3. Configure authorization if required in the **Authorization** section. The HTTP Webhook Connector supports the following authorization methods:
+
+- **Basic** - The incoming requests must contain an `Authorization` header that contains the word Basic word followed by a space and a base64-encoded string username:password.
```

Comment:
Do you mean "That contains the word `Basic`" here?

Reply:
Exactly! Good catch